src/main/java/co/omise/models/Charge.java

```diff
@@ -123,7 +123,11 @@ public void setTransaction(String transaction) {
     }
 
     public SourceOfFunds getSourceOfFund() {
-        return sourceOfFund;
+        if (sourceOfFund != null) {
+            return sourceOfFund;
+        } else {
+            return card != null ? SourceOfFunds.Card : SourceOfFunds.Offsite;
```

Comment:
I think we should fix the API-Version to the latest and ignore/remove this property

Reply:
We already fix the API version. As talked with support team we have to keep `sourceOfFunds` until next version.

Reply:
That would be even better actually

Reply:
This is a part of Omise Java V2?

Reply:
Yes

Reply:
Oh OK then